components/dropdown/user/src/index.js

```diff
@@ -39,14 +39,16 @@ const DropdownUser = ({
       'sui-DropdownUserMenu-listLinkHighlight': highlight
     })
 
+    const hasLinkNotifications = Boolean(notifications)
```

Comment:
just wondering... should the class `has-notifications` on the wrapper be now linked to `hasLinkNotifications`? is it of any use `hasNotifications` as a prop now?

https://github.com/SUI-Components/adevinta-spain-components/pull/373/files#diff-8499a52894be46a94b20c18fd8e8c7259a6eec84647174bf13e26d72e5af720bR65

Reply:
the **hasNotifications** prop it's just for manage the `wrapper class` to show a **badge notification** in the user avatar.